data_diff/diff_tables.py

```diff
@@ -290,8 +290,21 @@ def _bisect_and_diff_tables(self, table1, table2, info_tree):
         if key_type.python_type is not key_type2.python_type:
             raise TypeError(f"Incompatible key types: {key_type} and {key_type2}")
 
-        # Query min/max values
-        key_ranges = self._threaded_call_as_completed("query_key_range", [table1, table2])
+        if all([table1.min_key, table1.max_key, table2.min_key, table2.max_key]):
```

Comment:
You should test for `is None`, because keys can be 0.

Reply:
Ah true, will fix that